lib/colorls/flags.rb

```diff
@@ -35,8 +35,17 @@ def process
 
     private
 
-    def flag_given?(flags)
+    def flag_given?(flags, unclubbable=false)
```

Comment:
This `unclubbable` can be computed by the `flag_given?` method depending on if there is any flag in `flags` that don't start with `--` right?

Reply:
I already have a `--` check, but for example “-h” should not be clubbable with other flags, thus a bool is needed to specify what should be combined and what should not.

Reply:
Hmm, but why not have `colorls -lha` be like help menu + `colorls -la`?

Reply:
I guess I can do that, sure.